pkg/apis/pipeline/v1beta1/task_validation_test.go

```diff
@@ -798,27 +809,6 @@ func TestTaskSpecValidateError(t *testing.T) {
 			Message: fmt.Sprintf("The value type specified for these keys %v is invalid", []string{"key1"}),
 			Paths:   []string{"params.task.properties"},
 		},
-	}, {
-		name: "keys defined in properties are missed in default",
```

Comment:
we might still want to validate for taskruns with inline task specs that all the keys have values defined somewhere.

Reply:
It's under the valid test cases called valid params type explicit and the param name is myobjWithDefaultPartialKeys.

Reply:
see this [comment](https://github.com/tektoncd/pipeline/pull/5427#discussion_r967250399)